.github/bin/update-os-image.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -x
```

Comment:
More info: https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail

```suggestion
set -euxo pipefail
```